docs/source/getting_started/handling_time.ipynb

```diff
@@ -0,0 +1,634 @@
+    "    include_cutoff_time=False,\n",
+    ")\n",
+    "actual"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fef9a797",
+   "metadata": {},
+   "source": [
+    "Approximating Features by Rounding Cutoff Times\n",
+    "-----------------------------------------------\n",
+    "\n",
+    "For each unique cutoff time, Featuretools must perform operations to select the data that’s valid for computations. If there are a large number of unique cutoff times relative to the number of instances for which we are calculating features, the time spent filtering data can add up. By reducing the number of unique cutoff times, we minimize the overhead from searching for and extracting data for feature calculations.\n",
+    "\n",
+    "One way to decrease the number of unique cutoff times is to round cutoff times to an earlier point in time. An earlier cutoff time is always valid for predictive modeling — it just means we’re not using some of the data we could potentially use while calculating that feature. So, we gain computational speed by losing a small amount of information.\n",
+    "\n",
+    "To understand when an approximation is useful, consider calculating features for a model to predict fraudulent credit card transactions. In this case, an important feature might be, \"the average transaction amount for this card in the past\". While this value can change every time there is a new transaction, updating it less frequently might not impact accuracy."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "a718dff5",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "\n",
+    "    The bank BBVA used approximation when building a predictive model for credit card fraud using Featuretools. For more details, see the \"Real-time deployment considerations\" section of the `white paper <https://arxiv.org/pdf/1710.07709.pdf>`_ describing the work involved.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "328147f5",
+   "metadata": {},
+   "source": [
+    "The frequency of approximation is controlled using the ``approximate`` parameter to [featuretools.dfs](../generated/featuretools.dfs.html#featuretools.dfs) or [featuretools.calculate_feature_matrix](../generated/featuretools.calculate_feature_matrix.html#featuretools.calculate_feature_matrix). For example, the following code would approximate aggregation features at 1 day intervals::\n",
+    "\n",
+    "    fm = ft.calculate_feature_matrix(features=features,\n",
+    "                                     entityset=es_transactions,\n",
+    "                                     cutoff_time=ct_transactions,\n",
+    "                                     approximate=\"1 day\")\n",
+    "\n",
+    "In this computation, features that can be approximated will be calculated at 1 day intervals, while features that cannot be approximated (e.g \"what is the destination of this flight?\") will be calculated at the exact cutoff time.\n",
```

Comment:
Since this paragraph is on a fraud example kick, I think a better example feature would be "where did this transaction occur"

Reply:
makes sense to me, changing